### ISSUE TITLE
Pressing I (information) during watching a movie, only shows cover.

### ISSUE THREAD

**Issue by kraaa** [pix/aeon#114]
Bug or supposed to be? I wanna be able to read info on the movie and location if possible.

**Comment by sergiocos** [comment]
This is how it supposed to look: http://img191.imageshack.us/img191/1659/screenshot000l.png
There's no reason to clutter the info screen with unnecessary info. Plot may be read BEFORE you start the movie. And you can always stop the movie, read everything and resume.

**Comment by kraaa** [comment]
Well i guess this is just my personal preference :p

**Comment by phil65** [comment]
but i agree that the info screen should have more info than the seekbar.
so a) add more info to the info screen or b) reduce the info for seekbar

**Comment by kraaa** [comment]
How about something like this?: 
http://data.fuskbugg.se/skogsturken/screenshot000l.png
date with (2009) after title and popup plot/director info to the right.
Maybe codec infos at the top in the middle.

**Comment by ghost** [comment]
No please don't change this, why would you want plot in the seekbar?

Press tab and read everything in the fullscreen info instead, this is really unnecessary.

**Comment by sialivi** [comment]
I agree with frellAn, please don't clutter this screen. I don't need to know the plot of the movie I'm already watching. Hell, even the genre and year is stretching it as it is.

**Comment by kraaa** [comment]
well just add actors then or something, so if you see a random actor you can just pause and scroll down on actors and see if its really him.

**Comment by sialivi** [comment]
As frellAn already mentioned, you can already do that by pressing tab

**Comment by phil65** [comment]
i think it´s not even possible to show actors, that´s restricted to dialogvideoinfo.
should we make the infoscreen exactly like the seekbar?
as i said, I think there should be more info when you press "i" than in the seekbar.

**Comment by phil65** [comment]
... or at least the same amount

**Comment by pix** [comment]
What about unifying these dialogs ? Why the info dialog so different that the Seekbar dialog ?

What do you think phil ?

**Comment by sialivi** [comment]
I withdraw my objections. I must have been thinking about the pause screen when I responded earlier. Just keep the pause screen uncluttered (infact, I would like genre, first aired, written by, removed from that screen)

**Comment by pix** [comment]
What do you think, should we made the info dialog more like the info one, merge the two, there are different but bassicaly shows the same thing (try to fast forward and press I)

**Comment by phil65** [comment]
not really sure if we should make two identical windows or strip down dialogprogress a bit and use the actual dialogprogress as info. (or perhaps sliding up the dialogprogess a bit when pressing info with a bit more information revealed at some lines at the bottom of the screen? (or keeping the upper bar of the info screen?)

**Comment by Bat3man** [comment]
For unifiying purposes I think it's a good idea to "merge" these two. The infoscreen has less info than the pause/scroll screen.

I think it's a good idea to make them LOOK the same, but the info screen gives you a little more info. Maybe make these two AND the menu (with all the buttons) look the same.

For me: the info that I like (personal) and would make sense on an info overlay:

Poster, albeit maybe a little smaller. Or for tv shows: the thumb of the episode, or the logo.png (keeps it cleaner, more aeonish)
Timing (finish time, current time, and seekbar times.
Rating in stars
Director
Year

Some Cool animation would make it really slick!

A few weeks ago you guys had a new texture (black with a fade on the top), which had some lines in it for the timings, I liked that, but there must be a reason why it was removed again.

Good luck debating this further. ;-)

**Comment by Bat3man** [comment]
Come to think of it: the clearart makes it too much, Think the logo.png is much better, cooler. Just a thought.

**Comment by phil65** [comment]
i also prefer the logo.png. pix, what do you think?

**Comment by pix** [comment]
Add a config swich perhaps ? (I like the clearart), perhaps another to hide it completely ? What do you think ?

**Comment by sergiocos** [comment]
Yes, logos are much better than clearart (quality wise), but logos may be seen in some views, but for clearart, this is the only way to use it. If people want logos, there is a fallback to logo.png, they just have to delete the cleararts (all of them).
